app/models/concerns/orchestration/puppetca.rb

```diff
@@ -29,6 +29,12 @@ def delCertificate
   # Empty method for rollbacks - maybe in the future we would support creating the certificates directly
   def setCertificate; end
 
+  # Rest certname based on whether to use uuids or the hostname
+  def resetCertname
```

Comment:
Don't we usually use `snake_case`?

Reply:
We do, but not for methods used in the orchestration queue. I'd keep this for consistency.